jupyterhub/app.py

```diff
@@ -352,6 +352,13 @@ def _hub_prefix_changed(self, name, old, new):
     cookie_secret_file = Unicode('jupyterhub_cookie_secret',
         help="""File in which to store the cookie secret."""
     ).tag(config=True)
+    
+    api_tokens = Dict(Unicode(),
+        help="""Dict of token:username to be loaded into the database.
+        
+        Allows ahead-of-time generation of API tokens for use by services.
```

Comment:
@minrk I'm not sure the following is an improvement but "ahead-of-time" and "use by services" together felt a little vague to me.

Allows administrator to add pre-generated user API tokens for use by services.